src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,515 @@
+
+  NRQL, short for New Relic query language, is a powerful tool for accessing detailed data from New Relic. NRQL is like SQL. It lets you gather specific info about your applications, hosts, and business operations. By using NRQL, you can gain valuable insights into the performance and behavior of your environment. 
+
+  Learn more about dashboards [here](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/) and about NRQL [here](/docs/nrql/get-started/introduction-nrql-new-relics-query-language/).
+  
+  </Side>
+  </SideBySide>
+
+# Errors inbox [#errors-inbox]
+<SideBySide>
+  <Side>
+
+  Errors inbox is an error-tracking solution. It's designed to put you in control. It helps you find, focus on, and fix errors across your application stack. Errors inbox is embedded in the <InlinePopover type="apm" /> UI so you can resolve errors faster.
+
+  Learn more about errors inbox [here](/docs/apm/errors-inbox/errors-inbox-ui).
+
+  </Side>
+  <Side>
+
+  <img
+  title="New Relic Errors inbox"
+  alt="New Relic Errors inbox"
+  src={NRErrosInboxMain}
+  />
+
+  </Side>
+  </SideBySide>
+
+# IAST [#iast]
+<SideBySide>
+  <Side>
+
+  <img
+  title="New Relic IAST"
+  alt="New Relic IAST"
+  src={NRIastMain}
+  />
+
+  </Side>
+  <Side>
+
+  New Relic Interactive Application Security Testing (IAST) can help you stop cyberattacks and breaches on your applications. It does this by probing your running code for exploitable flaws.
+
+  Learn more about IAST [here](/docs/iast/introduction).
+
+  </Side>
+  </SideBySide>
+
+# Infrastructure [#infrastructure]
+<SideBySide>
+  <Side>
+
+  The <InlinePopover type="infrastructure" /> capability lets you monitor the health and performance of your service. It covers containers and cloud infrastructure. It includes tracking CPU, memory, network traffic, and disk use. It also includes over 400 on-host integrations for monitoring third-party apps. 
+
+  Learn more about infrastructure [here](/docs/infrastructure/infrastructure-monitoring/get-started/get-started-infrastructure-monitoring/).
+
+  </Side>
+  <Side>
+
+  <img
+    title="New Relic infrastructure"
+    alt="New Relic infrastructure"
+    src={NRInfraMain}
+  />
+  </Side>
+  </SideBySide>
+
+# Kubernetes and Pixie [#k8s-pixie]
+<SideBySide>
+  <Side>
+
+  <img
+    title="New Relic Kubernetes"
+    alt="New Relic Kubernetes"
+    src={NRK8sMain}
+  />
+
+  </Side>
+  <Side>
+
+  The New Relic Kubernetes integration gives you full observability. The data it provides lets you monitor the health of your whole Kubernetes cluster and the performance of your environment. You can check individual pods or drill into specific services and applications. The agent collects telemetry data from your cluster. It uses several New Relic integrations. These include Kubernetes events, the Prometheus Agent, and the New Relic Logs Kubernetes plugin.
```

Comment:
It uses several New Relic integrations. These include Kubernetes events, the Prometheus Agent, and the New Relic Logs Kubernetes plugin.

To:

It uses several New Relic integrations, including Kubernetes events, the Prometheus Agent, and the New Relic Logs Kubernetes plugin.